model/host/hostutil.go

```diff
@@ -238,7 +238,7 @@ func (h *Host) FetchAndReinstallJasperCommand(config evergreen.HostJasperConfig)
 // delete the current Jasper service configuration (if it exists), install the
 // new configuration, and restart the service.
 func (h *Host) ForceReinstallJasperCommand(config evergreen.HostJasperConfig) string {
-	params := []string{fmt.Sprintf("--port=%d", config.Port)}
+	params := []string{"--host=0.0.0.0", fmt.Sprintf("--port=%d", config.Port)}
```

Comment:
What does it mean to set host to 0.0.0.0 for jasper?

Reply:
If the host isn't specified, Jasper will listen on localhost by default so it's only reachable from connections from the same host. Using 0.0.0.0 allows it to listen on all available IPs for that host, which lets the app server connect to Jasper remotely using RPC.

Reply:
Doesn't that mean that it will listen potentially listen on ports that are open to the world?

Reply:
As far as I know, the firewall rules only allow traffic from the evergreen app servers in the port range 1024-2500.

Reply:
Sorry, totally missed that this applies to IP addresses, not to ports.